control/items_control.txt

```diff
@@ -72,242 +72,242 @@ Arrow 3000 1 0
 ##### WORTHLESS GEARS #####
 # Worthless gears which are sold by default
 
-2305 0 0 1 # Adventurer's Suit
-2211 0 0 1 # Bandana
-2217 0 0 1 # Biretta
-2405 0 0 1 # Boots
-2103 0 0 1 # Buckler
-2227 0 0 1 # Cap
-2314 0 0 1 # Chain Mail
-2233 0 0 1 # Circlet
-2309 0 0 1 # Coat
-2310 0 0 1 # Coat [1]
-2302 0 0 1 # Cotton Shirt
-Cotton Shirt [1] 0 0 1
+Adventurer's Suit 0 0 1 # 2305
```

Comment:
I think add name and itemID ? 

```
Adventurer's Suit 0 0 1 # 2305
2305 0 0 1 # Adventurer's Suit
```

Reply:
it is not good, because that way, if the armor has refinement level, or has a card, it will sell the same way, because the ID is still the same